### PR TITLE
refactor ReadMe file

### PR DESCRIPTION
**Pull request by Asmaathabet**
relates #205

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,27 @@
...
 
 People who are busy and can't find a therapist quickly and easily or who don't have enough money to ask therapists and get contact with them or who don't know the appropriate type of therapy.
     
-## **Solution** 
-
-Make app to facilitate user reaching to the appropirate therpay and connect with therapist At the right time and at with less costs.
+## **Problem Solution** :bulb:
 
-## **User Journey**  
+Make app to facilitate user reaching to the appropirate therpay and connect with therapist in the right time with less costs.
```
Comment (leggomuhgreggo) [inline README.md]:
```suggestion
Make app to facilitate user reaching to the appropriate therapy and connect with therapist in the right time with less costs.
```

File: README.md [hunk 2]
```diff
@@ -51,5 +38,84 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
 
-### **User who gives proper therapy to patients**
+### **User who gives proper therapy to patients** :syringe: 
 - As a therapist, I can fill out a form to create and submit a profile.
+
+## **User Journey**  :pencil2:
+
+### **User who want to take a therapy** :pill:
+
+As a user I can determine my status by answering a Questionnaire in order to find a specific therapist that I believe he/she will be appropriate for me and I can see list of Therapies according to questionnaire result, or I can choose to search (filter) for therapists by various categories like location, cost, specialty, or even by Skype/remote possibility Or I can visit a therapist profile to get more information about him/her, or learn more about therapists types, and I can see Glossary that explains scientific and strange words and I can move freely using the menu in the app, also I can get more about the app using the description that written in seprate about section.
```
Comment (leggomuhgreggo) [inline README.md]:
```suggestion
As a user I can determine my status by answering a Questionnaire in order to find a specific therapist that I believe he/she will be appropriate for me and I can see list of Therapies according to questionnaire result, or I can choose to search (filter) for therapists by various categories like location, cost, specialty, or even by Skype/remote possibility Or I can visit a therapist profile to get more information about him/her, or learn more about therapists types, and I can see Glossary that explains scientific and strange words and I can move freely using the menu in the app, also I can get more about the app using the description that written in separate about section.
```

File: README.md [hunk 3]
```diff
@@ -51,5 +38,84 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+    `npm run dev`
+
+    then you should be able to go to [localhost](http://localhost:5000/) 
+> To run the client side , In your terminal Type: 
+    `cd client`
+    `npm start`
+
+    then you will be able to run [localhost](http://localhost:3000/) and view the app live in the Browser!
+
+## **Rescources**  :mag:
```
Comment (leggomuhgreggo) [inline README.md]:
```suggestion
## **Resources**  :mag:
```

File: README.md [hunk 4]
```diff
@@ -51,5 +38,84 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+> To run the client side , In your terminal Type: 
+    `cd client`
+    `npm start`
+
+    then you will be able to run [localhost](http://localhost:3000/) and view the app live in the Browser!
+
+## **Resources**  :mag:
+* [Node Js](https://nodejs.org/en/)
+* [Express Framework](https://expressjs.com/)
+* [Joi library](https://www.npmjs.com/package/joi)
```
Comment (AbdallahAmmar96) [inline README.md]: We Yup instead of Joi!!

File: README.md [hunk 5]
```diff
@@ -51,5 +38,84 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+Database: **Airtable**  
+Styling:  **CSS**   
+Libraries: **AntDesign** & **Leaflet**
+
+## **Launch App**  :mega:
+
+> First clone this repo:  
+`git clone https://github.com/GSG-G7/shrinc.git`
+
+> Run `npm i` to install the dependencies for the app as general.
```
Comment (AbdallahAmmar96) [inline README.md]: `npm i` install the dependencies and dev dependencies

File: README.md [hunk 6]
```diff
@@ -51,5 +38,84 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+
+> Run `cd client` and `npm i` to install dependencies for client- React Js .
```
Comment (AbdallahAmmar96) [inline README.md]:
```suggestion
> Run `cd client` and `npm i` to install the packages for client- React Js . 
```

File: README.md (hunk 4 above)
Comment (Asmaathabet) [reply README.md]: I fixed it

File: README.md [hunk 7]
```diff
@@ -51,5 +38,84 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+Airtable_KEY
+Base_ID
+Cloud_Name
+Cloudinary_KEY
+Cloudinary_SECRET
+```
+
+### Start the App :electric_plug:
+
+To start the App Locally you have to start server First then start client side !
```
Comment (AbdallahAmmar96) [inline README.md]: The server should not be the first !!!

File: README.md [hunk 8]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+* You should login to [Cloudinary](https://cloudinary.com/) Account .
+* You will gain [Cloud_Name], [Cloudinary_KEY], [Cloudinary_SECRET] environmental variables needed to upload images to database.
+* Set the database variables in your .env file as follows
+
+### Environment Variables :key:
+Environment variables are one of the ways we keep our product safe. If you want to access our app locally you will need to add your own.
+
+First create a [.env](https://github.com/dwyl/env2#create-a-env-file) file in the App route and add the following variables:
+```
+Airtable_KEY
```
Comment (hashemt) [inline README.md]: This is a personal API key. It’s required in order to use the Airtable API.

File: README.md [hunk 9]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+BackEnd: **Node JS** & **Express JS**    
+FrontEnd: **React JS**  
+Database: **Airtable**  
+Styling:  **CSS**   
+Libraries: **AntDesign** & **Leaflet**
+
+## **Launch App**  :mega:
+
+> First clone this repo:  
+`git clone https://github.com/GSG-G7/shrinc.git`
```
Comment (AbdallahAmmar96) [inline README.md]:
```suggestion
> Or write this in your terminal:
`git clone https://github.com/GSG-G7/shrinc.git`
```

File: README.md [hunk 10]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+To start the App Locally you can start server First then start client side or vice versa!
+> To run the Server , In your terminal Type: 
+
+    `npm run dev`
+
+    then you should be able to go to [localhost](http://localhost:5000/) 
+> To run the client side , In your terminal Type: 
+    `cd client`
+    `npm start`
+
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
```
    cd client
    npm start
```
```

File: README.md [hunk 11]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+#### setup Airtable :triangular_ruler:
+* You should login to  [Airtable](https://airtable.com) Account to have access to its API .    
+* Click on Shrinc Workspace that it will redirect you to its Documentation API .    
+* In The Documentation you will find [Airtable_KEY ], [Base_ID] environmental variables needed to run app.
+* Set the database variables in your .env file as follows
+
+#### setup Cloudinary :star:
+* You should login to [Cloudinary](https://cloudinary.com/) Account .
+* You will gain [Cloud_Name], [Cloudinary_KEY], [Cloudinary_SECRET] environmental variables needed to upload images to database.
+* Set the database variables in your .env file as follows
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
* Set the database variables in your .env file as follows.
```

File: README.md [hunk 12]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+
+> Run `cd client` and `npm i` to install the packages for client- React Js . 
+
+### Database Setup  :clipboard:
+
+#### setup Airtable :triangular_ruler:
+* You should login to  [Airtable](https://airtable.com) Account to have access to its API .    
+* Click on Shrinc Workspace that it will redirect you to its Documentation API .    
+* In The Documentation you will find [Airtable_KEY ], [Base_ID] environmental variables needed to run app.
+* Set the database variables in your .env file as follows
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
* Set the database variables in your .env file as follows.
```

File: README.md [hunk 13]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+* You will gain [Cloud_Name], [Cloudinary_KEY], [Cloudinary_SECRET] environmental variables needed to upload images to database.
+* Set the database variables in your .env file as follows
+
+### Environment Variables :key:
+Environment variables are one of the ways we keep our product safe. If you want to access our app locally you will need to add your own.
+
+First create a [.env](https://github.com/dwyl/env2#create-a-env-file) file in the App route and add the following variables:
+```
+Airtable_KEY
+Base_ID
```
Comment (hashemt) [inline README.md]: This Base_ID to define which base you want to connect with it.

File: README.md [hunk 14]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+### Database Setup  :clipboard:
+
+#### setup Airtable :triangular_ruler:
+* You should login to  [Airtable](https://airtable.com) Account to have access to its API .    
+* Click on Shrinc Workspace that it will redirect you to its Documentation API .    
+* In The Documentation you will find [Airtable_KEY ], [Base_ID] environmental variables needed to run app.
+* Set the database variables in your .env file as follows
+
+#### setup Cloudinary :star:
+* You should login to [Cloudinary](https://cloudinary.com/) Account .
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
* You should login to [Cloudinary](https://cloudinary.com/) Account.
```

File: README.md [hunk 15]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+Base_ID
+Cloud_Name
+Cloudinary_KEY
+Cloudinary_SECRET
+```
+
+### Start the App :electric_plug:
+
+To start the App Locally you can start server First then start client side or vice versa!
+> To run the Server , In your terminal Type:
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
> To run the Server, In your terminal Type: 
```

File: README.md [hunk 16]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+
+### Start the App :electric_plug:
+
+To start the App Locally you can start server First then start client side or vice versa!
+> To run the Server , In your terminal Type: 
+
+    `npm run dev`
+
+    then you should be able to go to [localhost](http://localhost:5000/) 
+> To run the client side , In your terminal Type:
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
> To run the client side, In your terminal Type: 
```

File: README.md [hunk 17]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+
+> Run `npm i` to install the dependencies and dev dependencies for the app as general.
+
+> Run `cd client` and `npm i` to install the packages for client- React Js . 
+
+### Database Setup  :clipboard:
+
+#### setup Airtable :triangular_ruler:
+* You should login to  [Airtable](https://airtable.com) Account to have access to its API .    
+* Click on Shrinc Workspace that it will redirect you to its Documentation API .
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
* Click on Shrinc Workspace that it will redirect you to its Documentation API.    
```

File: README.md [hunk 18]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+`git clone https://github.com/GSG-G7/shrinc.git`
+
+> Run `npm i` to install the dependencies and dev dependencies for the app as general.
+
+> Run `cd client` and `npm i` to install the packages for client- React Js . 
+
+### Database Setup  :clipboard:
+
+#### setup Airtable :triangular_ruler:
+* You should login to  [Airtable](https://airtable.com) Account to have access to its API .
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
* You should login to  [Airtable](https://airtable.com) Account to have access to its API. 
```

File: README.md [hunk 19]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+Libraries: **AntDesign** & **Leaflet**
+
+## **Launch App**  :mega:
+
+> First clone this repo:  
+`git clone https://github.com/GSG-G7/shrinc.git`
+
+> Run `npm i` to install the dependencies and dev dependencies for the app as general.
+
+> Run `cd client` and `npm i` to install the packages for client- React Js .
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
> Run `cd client` and `npm i` to install the packages for client- React Js.
```

File: README.md [hunk 20]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+* Set the database variables in your .env file as follows
+
+### Environment Variables :key:
+Environment variables are one of the ways we keep our product safe. If you want to access our app locally you will need to add your own.
+
+First create a [.env](https://github.com/dwyl/env2#create-a-env-file) file in the App route and add the following variables:
+```
+Airtable_KEY
+Base_ID
+Cloud_Name
```
Comment (hashemt) [inline README.md]: The cloud name identifier is the name of your Cloudinary account, This name is assigned to your account during signup (the field can be edited) and is listed in the Account Details.

File: README.md [hunk 21]
```diff
@@ -51,5 +38,85 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+Cloudinary_KEY
+Cloudinary_SECRET
```
Comment (hashemt) [inline README.md]: Cloudinary auto-generates a pair of access identifiers, the API Key and API Secret, that need to be used for communicating with Cloudinary's API and signing your requests.
These identifiers are listed in the Account Details.

File: README.md [hunk 22]
```diff
@@ -51,5 +38,96 @@ As a therapist I can sign up in the app as a a therapy by filling the form that
...
+* In The Documentation you will find [Airtable_KEY ], [Base_ID] environmental variables needed to run app.
+* Set the database variables in your .env file as follows.
+
+#### setup Cloudinary :star:
+* You should login to [Cloudinary](https://cloudinary.com/) Account.
+* You will gain [Cloud_Name], [Cloudinary_KEY], [Cloudinary_SECRET] environmental variables needed to upload images to database.
+* Set the database variables in your .env file as follows.
+
+### Environment Variables :key:
+Environment variables are one of the ways we keep our product safe. If you want to access our app locally you will need to add your own.
```
Comment (MohammadAlhalaq) [inline README.md]:
```suggestion
Environment variables are one of the ways we keep our product safe, If you want to access our app locally you will need to add your own.
```

**Pull request merged**